src/Hosting.Services.Web/Middlewares/UserIdentity/IdentityProviders/EmailBasedUserIdentityProvider.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
+{
+	internal class EmailBasedUserIdentityProvider : IUserIdentityProvider
+	{
+        private static int MaxBytesPerChar { get; } = 4; // UTF-8
```

Comment:
The indentation for a lot of the lines is off, which makes the code difficult to read. Please fix this throughout.

I believe the issue is a mix of tabs and spaces. Please align with the rest of the project. To help, I'd advise turning on the setting in your IDE to make whitespace visible.

Reply:
The indentation is still strange. Take a look in GitHub and you'll see what I mean.

Reply:
This applies to all of the files

Reply:
Seems to be fixed by `dotnet format`, a lot of other files are updated as well... Not sure if I should dismiss those changes

Reply:
Looks good now, thanks!